docs/source/reference/router/telemetry/instrumentation/selectors.mdx

```diff
@@ -109,5 +110,110 @@ GraphQL metrics are extracted from the response data the router returns to clien
 | `field_name`     | No          | `string`         | The name of a field from the response data  |
 | `field_type`     | No          | `string`         | The type of a field from the response data  |
 | `type_name`      | No          |                  | The GraphQL type from the response data     |
-| `operation_name` | Yes         | `string`\|`hash` | The operation name of the query             |
+| `operation_name` | Yes         | `string` \| `hash` | The operation name of the query             |
 | `static`         | No          |                  | A static string value                       |
+
+
+## Example selector configurations
+
+Some example configurations of common use cases for selectors:
+
+### Configuring trace ID
+
+Getting the trace ID of a request with `trace_id`:
+
+```yaml title="router.yaml"
+telemetry:
+  instrumentation:
+    events:
+      router:
+        my.event:
+          message: 'An event occurred'
+          level: error
+          on: response
+          condition:
+            eq:
+              - on_graphql_error: true
+              - true
+          attributes:
+            # This key should match the trace ID from the request
+            id_from_header:
+              trace_id: open_telemetry
+```
+
+### Setting JSON paths
+
+Configuring selectors with JSON paths, such as supergraph `response_data` and `response_errors`:
```

Comment:
This will create a high cardinality metric:

A better example is this:
```
telemetry:
  exporters:
    metrics:
      prometheus:
        enabled: true
  instrumentation:
    instruments:
      supergraph:
        my.request.on_graphql_error:
          value: event_unit
          type: counter
          unit: error
          description: my description
          condition:
            exists:
              response_errors: "$.[0].extensions.code"
          attributes:
            response_errors:
              response_errors: "$.[0].extensions.code"
```